### ISSUE TITLE
Rule: no-ambiguity-target

### ISSUE THREAD

**Issue by igorkamyshev** [effector/eslint-plugin#19]
This code:
```ts
const $someData = createStore(null)
const someEvent = createEvent()
const targetEvent = createEvent()

const otherEvent = sample({ source: $someData, clock: someEvent, target: targetEvent });
```
is ambiguity, but in real world it means
```ts
const $someData = createStore(null)
const someEvent = createEvent()
const targetEvent = createEvent()

sample({ source: $someData, clock: someEvent, target: targetEvent });

const otherEvent = targetEvent;
```

So, we should show a warning for this pattern.
